### PR TITLE
Adjustable statistics interval

### PR DESCRIPTION
**Pull request by phonedph1**
### Short description
Quick fix for #3781

Not sure if this should say "at most" since it's dependent on amount of queries coming in.

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled and tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: pdns/pdns_recursor.cc [hunk 1]
```diff
@@ -2061,7 +2061,7 @@ static void houseKeeping(void *)
     }
 
     if(!t_id) {
-      if(now.tv_sec - last_stat >= 1800) {
+      if(now.tv_sec - last_stat >= ::arg().asNum("statistics-interval") && ::arg().asNum("statistics-interval") != 0) {
```
Comment (pieterlexis) [inline pdns/pdns_recursor.cc]: `.asNum()` is pretty expensive. Perhaps a `g_statisticsInterval` would be better?

File: docs/markdown/recursor/settings.md [hunk 2]
```diff
@@ -93,6 +93,13 @@ have to tick an 'RFC 2181 compliant' box.
 Zones read from these files (in BIND format) are served authoritatively. DNSSEC is not supported. Example:
 `auth-zones=example.org=/var/zones/example.org, powerdns.com=/var/zones/powerdns.com`.
 
+## `statistics-interval`
+* Integer
+* Default: 1800
+* Available since: 4.1.0
+
+Interval between logging statistical summary on recursor performance.
```
Comment (Habbie) [inline docs/markdown/recursor/settings.md]: Can you add 0 to disable here as well?

**Comment by Habbie** [comment]
One nit, approved after that. Can you fix the nit and rebase?

**Comment by phonedph1** [comment]
Oh my goodness. I'll re-open this. I managed to get myself into git merge/rebase hell.

**Pull request closed without merging**